src/JNI/com/amazonaws/kinesis/video/producer/jni/Parameters.cpp

```diff
@@ -225,6 +226,20 @@ BOOL setClientInfo(JNIEnv *env, jobject clientInfo, PClientInfo pClientInfo) {
         CHK_JVM_EXCEPTION(env);
     }
 
+    methodId = env->GetMethodID(cls, "getAutomaticStreamingFlags", "()I");
+    if (methodId == NULL) {
+        DLOGW("Couldn't find method id getAutomaticStreamingFlags");
+    } else {
+        automaticStreamingFlag = env->CallIntMethod(clientInfo, methodId);
+    if (automaticStreamingFlag == AUTOMATIC_STREAMING_ALWAYS_CONTINUOUS) {
```

Comment:
can we simplify ? indentation does not seem right

Reply:
Fixed the indentation in the latest commit. The field `pClientInfo->automaticStreamingFlags` is an enum, sending and receiving enums in JNI may complicate it more is why I have used int instead.